src/scripts/json-presets-to-new-format.ts

```diff
@@ -0,0 +1,37 @@
+import { DatabaseContext, getNewConnection } from "../database_context";
+import _logger from "../logger";
+
+const logger = _logger("json-presets-to-new-format");
+
+async function exportJsonPresetsToNewTable(db: DatabaseContext): Promise<void> {
+    const jsonPresets = await db.kmq("game_option_presets_json").select("*");
+    await Promise.all(jsonPresets.map(async (preset) => {
+        const guildID = preset["guild_id"];
+        const presetName = preset["preset_name"];
+        const options = JSON.parse(preset["game_options"]);
+        try {
+            await Promise.all(
```

Comment:
This will throw an exception if _any_ of the promises fail. Should use `Promise.allSettled` and log the failed ones specifically

Reply:
For a preset, if any of its option insertions fail, it should stop inserting without attempting to insert the remaining options

Reply:
This should be a transaction then